smash.cabal

```diff
@@ -35,31 +35,31 @@ library
     cpp-options:       -DTESTING_MODE
 
   exposed-modules:
-      Lib
-      Offline
-      FetchQueue
-      FetchQueue.Retry
-    , Types
-    , DB
-    , DbSyncPlugin
+      Cardano.SMASH.Lib
```

Comment:
Some lines have `,` others don't, I suggest to be consistent here. `cabal-fmt` is also an interesting tool.

Reply:
Used `cabal-fmt`, even though I prefer manual formatting and commenting.